recipes/stages/instance-segmentation/imbalance.py

```diff
@@ -6,10 +6,6 @@
 
 task = 'instance-segmentation'
 
-data = dict(
```

Comment:
Do you have specific reason to remove this code? That can be needed to support semi-supervised learning.